open_spiel/games/dark_chess.cc

```diff
@@ -80,13 +81,57 @@ std::array<bool, chess::kMaxBoardSize * chess::kMaxBoardSize> ComputeObservabili
       auto &piece = board.at(sq);
       if (piece.color == color) {
         size_t index = chess::SquareToIndex(sq, board_size);
-        observability_table[index] = true;
+        if (!publicInfoTable[index]) observability_table[index] = true;
       }
     }
   }
   return observability_table;
 }
 
+// Computes which squares are common knowledge. It does not recognize all of them. Only squares of two opponent
```

Comment:
Please use the term "public information", "common knowledge" is a concept where players can distinguish finer information also based on the history of the play, i.e. "common knowledge" \subseteq "public information" : See https://arxiv.org/pdf/1906.11110.pdf Section 2.1 for more info.

We are not computing common knowledge here.